### ISSUE TITLE
how to show invalid Emils with a different font

### ISSUE THREAD

**Issue by bhedayatian** [OfficeDev/office-ui-fabric-react#10337]
Hello, I am trying to customize the people picker behaviour to show invalid Emails with a different font or color. ( for example in red)
this is how I create persona object dynamically 

const imageInitials='i', 
const secondaryText='test@yahoo';
const primaryText='test@yahoo';
const state=ValidationState.invalid;
 const persona = { imageInitials, secondaryText, primaryText, state ,
        styles: {
                primaryText: 
                  {color:'red'}   
               }
}

basically I need to overwrite .ms-Persona-primaryText color for invalid emails only

it doesn't do anything. any tips?  thanks

**Comment by KevinTCoughlin** [comment]
@bhedayatian, can you provide a Codepen repro that I may modify to unblock you per our issue template?

**Comment by bhedayatian** [comment]
@KevinTCoughlin
https://codesandbox.io/s/basic-floating-picker-0in8c

sorry, I wasn't extract all my code properly, the project doesn't run. if you look at the customPeoplePicker and where I created persona object you get the idea what I am trying to do. I just want to  change the font to red. thanks for your help. I have been stuck with this a couple of days :(

**Comment by bhedayatian** [comment]
@KevinTCoughlin 
I was also wondering why when I create a persona with ValidationState.invalid it doesn't add any special style to the persona to make it obvious to  user that its invalid email?

**Comment by bhedayatian** [comment]
@micahgodbolt @KevinTCoughlin any tips?

**Comment by KevinTCoughlin** [comment]
The reason the merge styles approach you have in the original issue report does not work for you is because merge styles is supported in `office-ui-fabric-react >= 6.0.0` versions.

I need to spend some time to see if this is at all possible in the most recent version. I am less familiar with the "Picker" code than @joschect.

**Comment by bhedayatian** [comment]
@KevinTCoughlin in the example that works https://codepen.io/zhangcathy/pen/OKXgEX
if you change the office ui version to 5.x.x it still works.. I am not sure why in my project styles don't work. I am exactly following the code from https://codepen.io/zhangcathy/pen/OKXgEX

**Comment by xugao** [comment]
@bhedayatian - are you still having this issue?
if so, can you pls confirm:
1) what is the fabric version you having issue with
2) what is the code you having issue with
3) are you not able to repro it in a codepen but only in your app?
